GitTfs/Program.cs

```diff
@@ -40,15 +44,15 @@ private static void ReportException(Exception e)
             if(gitTfsException != null)
             {
                 Trace.WriteLine(gitTfsException);
-                Console.WriteLine(gitTfsException.Message);
+                Trace.TraceError(gitTfsException.Message);
                 if (gitTfsException.InnerException != null)
                     ReportException(gitTfsException.InnerException);
                 if (!gitTfsException.RecommendedSolutions.IsEmpty())
                 {
-                    Console.WriteLine("You may be able to resolve this problem.");
+                    Trace.TraceError("You may be able to resolve this problem.");
                     foreach (var solution in gitTfsException.RecommendedSolutions)
                     {
-                        Console.WriteLine("- " + solution);
+                        Trace.TraceError("- " + solution);
```

Comment:
Is there any chance that nlog won't be configured to show errors by this point? For example, if `GetLogger()` throws an exception, maybe because the log file can't be written to, what will happen here? I'd like to err on the side of these messages always being shown.